src/libexpr/primops.cc

```diff
@@ -1664,9 +1664,28 @@ static void prim_findFile(EvalState & state, const PosIdx pos, Value * * args, V
     v.mkPath(state.checkSourcePath(state.findFile(searchPath, path, pos)));
 }
 
-static RegisterPrimOp primop_findFile(RegisterPrimOp::Info {
+static RegisterPrimOp primop_findFile(PrimOp {
     .name = "__findFile",
     .arity = 2,
+    .doc = R"(
+      Resolve the given key in a "search path" list.
```

Comment:
```suggestion
      Resolve the given key in a list of search paths.
```

Reply:
I am slightly worried about calling the elements of the list search paths, because they look like this:
```
nix-instantiate --eval -E 'builtins.nixPath'
[
  { path = "/etc/nixos/nixos-configuration/dep/nixpkgs"; prefix = "nixos"; }
  { path = "/etc/nixos/nixos-configuration/dep/nixpkgs"; prefix = "nixpkgs"; }
  { path = "/nix/var/nix/profiles/per-user/root/channels/nixos"; prefix = "nixpkgs"; }
  { path = "/etc/nixos/configuration.nix"; prefix = "nixos-config"; }
  { path = "/nix/var/nix/profiles/per-user/root/channels"; prefix = ""; }
]
```
do we want to call each of these attribute sets "search paths"?

I was thinking "search path" in the sense of `PATH` and `NIX_PATH`. But yes It is also confusing to call a list of paths a path though I agree that is some old Unix bullshit.

Reply:
Then give them a distinct-enough name and show the example. We could call them "search path records".